privacyidea/api/token.py

```diff
@@ -379,13 +381,13 @@ def list_api():
     if ufields:
         user_fields = [u.strip() for u in ufields.split(",")]
 
-    # filterRealm determines, which realms the admin would be allowed to see
-    filterRealm = ["*"]
-    # TODO: Userfields
+    if hasattr(request, "filterRealm"):
+        # filterRealm determines, which realms the admin would be allowed to see
+        filterRealm = request.filterRealm
+    else:
+        # In certain cases like for users, we do not have filterRealms  
+        filterRealm = None
```

Comment:
If you prefer, we could use ``filterRealm = getattr(request, "filterRealm", None)``. But we can also keep it like it above.

Reply:
Sounds good. Usually I do not program things, I do not understand! ;-)

Reply:
done in 80747ff6